### ISSUE TITLE
feature request: check if script references are valid

### ISSUE THREAD

**Issue by moritzjacobs** [peerigon/scriptlint#14]
For example:

```json
"scripts": {
    "test": "run-s test:unit lint",
    "test:unit": "jest",
    "test:lint": "eslint ./src"
}
```
should throw a scriptlint error because the script was renamed from `lint` to `test:lint` but the user forgot to change the reference in the `test` script as well.

This should be smart enough to go for `npm run <foo>`, `yarn <foo>`, and all variants of `npm-run-all`/`run-p` etc.
